portfolio/src/main/webapp/script.js

```diff
@@ -15,14 +15,16 @@
 /**
```

Comment:
Js docs are required for js files too. example : 
/**
  * This is the main javascript file for the page. Contains a blah blah method.
  * @author: sadreactsonly
  */

Also, move all the existing Google LLC comments in this file description itself and put author name at the end i.e. 
/**
  * This is the main javascript file for the page. Contains a blah blah method.
  - leave one line - 
  Google LLC comments
  * @author: sadreactsonly
  */